BushBlitz/ev-training-exercise
# Feature/evobe 140/create reactjs app

### Ticket
https://phunts.atlassian.net/browse/EVOBE-140
https://phunts.atlassian.net/browse/EVOBE-144
https://phunts.atlassian.net/browse/EVOBE-145

### Changes
- Setup ReactJS Project
- Created Employee List Component
- Created Employee View Component
- Setup routing
- GET Employees API Integration

### Steps to Test

#### Test 1
1. Pull branch
2. deploy `etx-back-end`
3. navigate to `etx-front-end`
4. configure `.env` attribute `REACT_APP_SERVER_URL` to your own IP address (leave the port number)
5. ` npm install`
6. `npm start`
7.  navigate to `localhost:3000/view`
8. you should be able to view all employees 
9. click one row
10. it should display the selected employee

### Resources
None
